docker/base-builder/Dockerfile

```diff
@@ -22,24 +22,16 @@ RUN apt-get update -y && apt-get install -y \
     automake \
     cmake \
     git \
-    golang \
```

Comment:
Is it because these are in base image ?

Reply:
That's probably the reason for some of them. TAnother reason is because we removed some benchmarks so we don't need their deps. And a last reason could be that some of these may not have been necessary to begin with.